packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -85,22 +88,31 @@ export class RelayerConfig {
       RATE_MODELS,
       CHAIN_IDS,
       L2_BLOCK_LOOKBACK,
+      CROSS_DOMAIN_FINALIZATION_THRESHOLD,
       RELAYER_ENABLED,
       FINALIZER_ENABLED,
       DISPUTER_ENABLED,
+      CROSS_DOMAIN_FINALIZER_ENABLED,
       WHITELISTED_CHAIN_IDS,
       L1_DEPLOY_DATA,
       L2_DEPLOY_DATA,
     } = env;
 
+    assert(BRIDGE_ADMIN_ADDRESS, "BRIDGE_ADMIN_ADDRESS required");
+    this.bridgeAdmin = toChecksumAddress(BRIDGE_ADMIN_ADDRESS);
+
     this.botModes = {
       relayerEnabled: RELAYER_ENABLED ? Boolean(RELAYER_ENABLED) : false,
       disputerEnabled: DISPUTER_ENABLED ? Boolean(DISPUTER_ENABLED) : false,
       finalizerEnabled: FINALIZER_ENABLED ? Boolean(FINALIZER_ENABLED) : false,
+      l2FinalizerEnabled: CROSS_DOMAIN_FINALIZER_ENABLED ? Boolean(CROSS_DOMAIN_FINALIZER_ENABLED) : false,
```

Comment:
See fix [here](https://github.com/UMAprotocol/protocol/pull/3593/files), this is eventually going to have to change to `CROSS_DOMAIN_FINALIZER_ENABLED === "true" ? true : false`